core/src/main/scala/org/locationtech/rasterframes/expressions/transformers/ReprojectGeometry.scala

```diff
@@ -49,12 +49,12 @@ import org.locationtech.rasterframes.model.LazyCRS
     > SELECT _FUNC_(geom, srcCRS, dstCRS);
        ..."""
 )
-case class ReprojectGeometry(geometry: Expression, srcCRS: Expression, dstCRS: Expression) extends Expression with CodegenFallback {
-
+case class ReprojectGeometry(geometry: Expression, srcCRS: Expression, dstCRS: Expression) extends TernaryExpression with CodegenFallback {
   override def nodeName: String = "st_reproject"
+  def first: Expression = geometry
+  def second: Expression = srcCRS
+  def third: Expression = dstCRS
```

Comment:
Couple of places the Expression case classes actually have meaningful names on the constructor args which feels nice. Where they don't the naming convention changed.